### PR TITLE
The summary screen of a request is not displayed V2

### PR DESCRIPTION
**Pull request by marcoAntonioNina**
## Issue & Reproduction Steps
When there are too many tokens in a request, the information cannot be opened.

## Solution
- The way information is obtained from closed forms has been changed

## How to Test
Steps in ticket

## Related Tickets & Packages
- [FOUR-16639](https://processmaker.atlassian.net/browse/FOUR-16639)
## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

[FOUR-16639]: https://processmaker.atlassian.net/browse/FOUR-16639?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-fef7aa81b5.engk8s.processmaker.net

File: ProcessMaker/Http/Controllers/Api/ProcessRequestController.php [hunk 1]
```diff
@@ -739,4 +740,36 @@ public function getRequestToken(Request $httpRequest, ProcessRequest $request)
 
         return new ApiResource($token);
     }
+
+    public function screenRequested(Request $httpRequest, ProcessRequest $request)
```
Comment (caleeli) [inline ProcessMaker/Http/Controllers/Api/ProcessRequestController.php:744]: Add PHPDOC for the method, and strict return type if possible
```
/**
 * Retrieve the screens requested for a given process request.
 *
 * @param  Request  $httpRequest
 * @param  ProcessRequest  $request
 *
 * @return ApiCollection
 */
 ```

File: resources/js/requests/components/RequestScreens.vue [hunk 2]
```diff
@@ -93,7 +94,9 @@
         ]
       };
     },
-
+    mounted() {
+      this.fetch();
```
Comment (caleeli) [inline resources/js/requests/components/RequestScreens.vue:98]: Does the RequestScreens show a loading state while the data is being fetched?

File: routes/api.php [hunk 3]
```diff
@@ -220,6 +220,7 @@
     Route::delete('requests/{request}', [ProcessRequestController::class, 'destroy'])->name('requests.destroy')->middleware('can:destroy,request');
     Route::get('requests/{request}/tokens', [ProcessRequestController::class, 'getRequestToken'])->name('requests.getRequestToken')->middleware('can:view,request');
     Route::post('requests/{request}/events/{event}', [ProcessRequestController::class, 'activateIntermediateEvent'])->name('requests.update,request');
+    Route::get('requests/{request}/details-screen-request', [ProcessRequestController::class, 'screenRequested'])->name('requests.detail.screen');
```
Comment (caleeli) [inline routes/api.php:223]: Consider to add the permission to view the request
```
->middleware('can:view,request')
```

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-fef7aa81b5.engk8s.processmaker.net

**Review by caleeli — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-fef7aa81b5.engk8s.processmaker.net

**Review by boliviacoca — APPROVED** [review]

**Pull request merged**